admin/static/js/metrics/metrics.js

```diff
@@ -236,7 +237,11 @@ var getWeeklyUserGain = function() {
 
 };
 
-var renderKeenMetric = function(element, type, query, height, colors=null, keenClient=client) {
+var renderKeenMetric = function(element, type, query, height, colors, keenClient) {
+
+    if (!keenClient) {
```

Comment:
Is this going to work if nothing is passed in? My memory of JS is that you have to `typeof keenClient != "undefined"` or some other nonsense.